.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: ci
+
+on:
+  pull_request:
+
+jobs:
+  static-analysis:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

Comment:
This might give us a clue what this is installing: https://github.com/actions/setup-python/blob/main/src/install-python.ts

Mostly my interest is how can we recreate this image locally. But this is nonblocking, more of a curiosity to see if we can avoid building our own docker image.